machine/lcd.ino

```diff
@@ -0,0 +1,71 @@
+// include the library code:
+#include <LiquidCrystal.h>
+
+// initialize the library with the numbers of the interface pins
+LiquidCrystal lcd(19, 23, 18, 17, 16, 15);
+
+
+
+void displayStuffOnLCD(void *pvParameters) {
+
+  while (true) {
+    EventBits_t uxBits = xEventGroupWaitBits(
+                           updateLcdEventGroup,   /* The event group being tested. */
+                           BIT_UPDATE_DISPLAY | BIT_WAITING_RFID, /* The bits within the event group to wait for. */
+                           pdTRUE,        /* bits cleared before returning. */
+                           pdFALSE,       /* Don't wait for both bits, either bit will do. */
+                           portMAX_DELAY
+                         );
```

Comment:
Is there some "wait" or delayed behavior in these functions? If it isn't necessary to run this functon ever time this `while(true)` could be debounced to don't waste a lot of clock cycles in this function